components/BottomSheet/src/MDCBottomSheetPresentationController.h

```diff
@@ -64,6 +64,11 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ This is used to set a custom height on the sheet view.
+ */
+@property(nonatomic) CGFloat preferredSheetHeight;
```

Comment:
Please mark this `nonatomic, assign` to match the file style.

Reply:
Done